Add Podcast to social icon options

B77Mills opened the pull request:
This is added support for when the cms allows it https://github.com/parameter1/base-platform/pull/188

<img width="1074" alt="Screen Shot 2022-12-16 at 9 26 15 AM" src="https://user-images.githubusercontent.com/3845869/208131544-c892ce30-226b-42f6-98c6-26494f5f6546.png">

B77Mills committed:
Add Podcast to social icon options

B77Mills committed:
Add podcast icon to marko-web-icon package

B77Mills committed:
Use podcast icon instead of mic

solocommand reviewed (changes requested):
This should not be a generic icon, it should be for a specific provider such as Megaphone or [Anchor.fm](http://anchor.fm/).

solocommand commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Base enhancement - podcast option on company pages](https://trello.com/c/Xbz2o35J/322-base-enhancement-podcast-option-on-company-pages)

B77Mills committed:
Replace generic podcast with anchor icon

Istead of adding a generic podcast icon we opted to add the anchor specific icon
